jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetSamProjectGenerator.kt

```diff
@@ -246,6 +233,7 @@ class DotNetSamProjectGenerator(
     private fun initProjectTextField() {
         projectNameField.text = SAM_HELLO_WORLD_PROJECT_NAME
         projectNameField.isEnabled = false
+        projectNameSetByUser = true
```

Comment:
assuming `changeProjectName` actually became `projectNameSetByUser`?

Reply:
Seems so, I could not find that flag anywhere in the parent class.